tests/PolynomialTest.java

```diff
@@ -55,4 +68,56 @@ void isSimplePolynomial() {
         assertTrue(PolynomialTools.isPolynomial(test));
     }
 
+    @Test
```

Comment:
Really good tests, get rid of this indent though

Reply:
wtf are you talking about, everything is indented properly.
![image](https://user-images.githubusercontent.com/61028214/108637970-046c8980-745b-11eb-8008-72465dd2dede.png)

Reply:
if this is github being weird thatn idc and I'm leaving it as is

Reply:
I'm pretty sure code review comments are visible to the public, just a heads-up ;)
Also yes according to your screenshot it's just github being weird